src/Symfony/Component/Workflow/Registry.php

```diff
@@ -41,7 +41,7 @@ public function has(object $subject, string $workflowName = null): bool
     /**
      * @return Workflow
      */
-    public function get(object $subject, string $workflowName = null)
+    public function get($subject, string $workflowName = null)
```

Comment:
This is a BC break as the class is not `final`.

Reply:
Still a BC break.

Reply:
oups, I deleted my comment unintentionally.

Anyway, I'm new to this. I don't understand what kind of path you want me to take.

Should I, for example :  
1. create another method
2. deprecate this one

And that case, how can I have the consistency I am looking for (which only goes through this BC Breaks) ?